projects/distributed-tracing/src/shared/dashboard/widgets/span-detail/data/span-detail-data-source.model.ts

```diff
@@ -54,6 +63,14 @@ export class SpanDetailDataSourceModel extends GraphQlDataSourceModel<SpanDetail
       tags: span.spanTags as Dictionary<unknown>
     };
   }
+
+  protected getTimeRangeOrThrow(): GraphQlTimeRange {
```

Comment:
Should we do this SpanGraphQlQueryHandlerService as well? We are not using it at many places.

Reply:
I think it's worth doing, but can happen in a separate PR

Reply:
will put  up a separate PR